content/module-reference/processing-modules/composite.md

```diff
@@ -1,7 +1,38 @@
 ---
-title: composite (placeholder)
+title: composite
 id: composite
 weight: 10
 applicable-version: 4.8
 ---
 
+Drag and drop an image from the film strip onto the "drom image from filmstrip here" box to have the image overlayed ontop of the currently edited image. Then adjust the various placement and adjustment attributes of the image being overlayed.
```

Comment:
Could do with an initial brief description of the purpose of the module at the start without getting into implementation details. Perhaps something like "Overlay an already-processed image to create a composite of the two images." I'm not entirely satisfied with that sentence but it illustrates what I mean. Most modules have this initial description and it's useful as a quick reference.

Reply:
Done.